public/stylesheets/custom.css

```diff
@@ -0,0 +1,5 @@
+@media (min-width: 980px) {
+  .upper-header {
+    padding-top: 50px;
```

Comment:
This is actually supposed to be attached to the Body of the page, because of the float of the Navbar